101-create-ase-with-webapp/azuredeploy.json

```diff
@@ -0,0 +1,280 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+  "parameters": {
+    "ASE-APP-SERVICE-APPWorkerSize": {
+      "type": "string",
+      "allowedValues": [
+        "0",
+        "1",
+        "2"
+      ],
+      "metadata": {
+        "description": "App Service Worker Size"
+      },
+      "defaultValue": "0"
+    },
+    "ASE-Location": {

```

Comment:
is there a particular reason you have the ASE location as a parameter? We generally recommend using "[resourceGroup().location]"